dashboard/app/views/levels/editors/_gamelab.html.haml

```diff
@@ -6,19 +6,11 @@
 
 = render partial: 'levels/editors/fields/debugger', locals: {f: f}
 
-= render partial: 'levels/editors/fields/control_buttons', locals: {f: f}
-
-= render partial: 'levels/editors/fields/submittable', locals: {f: f}
-
-= render partial: 'levels/editors/fields/key_value_data', locals: {f: f}
-
-.field.aligned-input-group
-  = boolean_check_box f, :hide_view_data_button
-  = f.label 'Hide View Data Button'
-
-= render partial: 'levels/editors/fields/validation_code', locals: {f: f}
-
-= render partial: 'levels/editors/fields/animation', locals: {f: f}
+%legend.control-legend.collapsed{data: {toggle: "collapse", target: "#display-area"}}
+  Display Area
+#display-area.in.collapse
+  = render partial: 'levels/editors/fields/control_buttons', locals: {f: f}
+  = render partial: 'levels/editors/fields/animation', locals: {f: f}
```

Comment:
now I am a bit confused. "animations" don't really seem like part of the display / visualization area. are these included here because the tabs to switch over to animations are near the visualization area? generally I'm feeling like the number of items here is small and the similarity is a bit low, so I'm not 100% sold on grouping these items together unless there are going to be more similar things added here in the future.

Reply:
Based on your feedback I ungrouped these fields.